app/src/main/res/layout/vocable_confirmation_dialog.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="@dimen/dialog_width"
+    android:layout_height="wrap_content"
+    android:background="@android:color/white"
+    android:elevation="@dimen/dialog_shadow">
+
+    <TextView
+        android:id="@+id/dialog_title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dialog_text_margin"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_text_margin"
+        android:textColor="@color/colorPrimaryDark"
+        android:textSize="@dimen/dialog_title_text_size"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Leaving the app" />
+
+    <TextView
+        android:id="@+id/dialog_message"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_text_margin"
+        android:textColor="@color/colorPrimaryDark"
+        android:textSize="@dimen/dialog_title_text_size"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="@+id/dialog_title"
+        app:layout_constraintTop_toBottomOf="@+id/dialog_title"
+        tools:text="You're about to be taken outside of the app. You may lose head tracking control." />
+
+    <com.willowtree.vocable.customviews.ActionButton
+        android:id="@+id/dialog_positive_button"
+        android:layout_width="@dimen/dialog_button_width"
+        android:layout_height="@dimen/dialog_button_height"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_button_end_margin"
+        android:layout_marginBottom="@dimen/dialog_button_end_margin"
+        android:background="@drawable/vocable_dialog_button_background"
+        android:gravity="center"
+        android:textAllCaps="true"
```

Comment:
This is true by default I think? (if `ActionButton` is a subclass of `Button`/`AppCompatButton`)

Reply:
It's a subclass of `AppCompatButton`, but for some reason the text isn't caps unless that property is in there.